homeassistant/components/luxtronik/const.py

```diff
@@ -0,0 +1,180 @@
+"""Constants for the Luxtronik integration."""
+# region Imports
+from datetime import timedelta
+from enum import Enum
+import logging
+from typing import Final
+
+from homeassistant.const import (
+    DEVICE_CLASS_ENERGY,
+    DEVICE_CLASS_PRESSURE,
+    DEVICE_CLASS_TEMPERATURE,
+    DEVICE_CLASS_TIMESTAMP,
+    ELECTRIC_POTENTIAL_VOLT,
+    ENERGY_KILO_WATT_HOUR,
+    PERCENTAGE,
+    PRESSURE_BAR,
+    TEMP_CELSIUS,
+    TEMP_KELVIN,
+    TIME_HOURS,
+    TIME_SECONDS,
+)
+
+# endregion Imports
+
+# region Constants Main
+DOMAIN: Final = "luxtronik"
+
+LOGGER: Final[logging.Logger] = logging.getLogger(__package__)
+
+PLATFORMS: list[str] = ["sensor"]
+# endregion Constants Main
+
+# region Conf
+CONF_SAFE: Final = "safe"
+CONF_LOCK_TIMEOUT: Final = "lock_timeout"
+CONF_UPDATE_IMMEDIATELY_AFTER_WRITE: Final = "update_immediately_after_write"
+
+CONF_PARAMETERS: Final = "parameters"
+CONF_CALCULATIONS: Final = "calculations"
+CONF_VISIBILITIES: Final = "visibilities"
+
+CONF_LANGUAGE_SENSOR_NAMES: Final = "language_sensor_names"
+
+DEFAULT_PORT: Final = 8889
+
+LANG_EN: Final = "en"
+LANG_DE: Final = "de"
+LANG_DEFAULT: Final = LANG_EN
+LANGUAGES: Final = Enum(LANG_EN, LANG_DE)
+LANGUAGES_SENSOR_NAMES: Final = [LANG_EN]
+# endregion Conf
+
+
+ATTR_STATUS_TEXT: Final = "status_text"
+
+
+MIN_TIME_BETWEEN_UPDATES: Final = timedelta(seconds=10)
+
+
+# region Lux Status
+LUX_STATUS_HEATING: Final = "heating"  # 0
+LUX_STATUS_DOMESTIC_WATER: Final = "hot water"  # 1
+LUX_STATUS_SWIMMING_POOL_SOLAR: Final = "swimming pool/solar"  # 2
+LUX_STATUS_EVU: Final = "evu"  # 3
+LUX_STATUS_DEFROST: Final = "defrost"  # 4
+LUX_STATUS_NO_REQUEST: Final = "no request"  # 5
+LUX_STATUS_HEATING_EXTERNAL_SOURCE: Final = "heating external source"  # 6
+LUX_STATUS_COOLING: Final = "cooling"  # 7
+
+LUX_STATUS_NONE: Final = "None"
+LUX_STATUS_UNKNOWN: Final = "unknown"
+
+LUX_STATUS1_HEATPUMP_IDLE: Final = "heatpump idle"
+LUX_STATUS1_PUMP_FORERUN: Final = "pump forerun"
+LUX_STATUS1_HEATPUMP_COMING: Final = "heatpump coming"
+
+LUX_STATUS3_GRID_SWITCH_ON_DELAY: Final = "grid switch on delay"
+
+LUX_STATES_ON: list[str] = [
```

Comment:
Does it seem like the order matters? If so, this would be a mapping.. shouldn't this be a dict?